synapse/config/emailconfig.py

```diff
@@ -91,10 +100,15 @@ def default_config(self, config_dir_path, server_name, **kwargs):
         # Defining a custom URL for Riot is only needed if email notifications
         # should contain links to a self-hosted installation of Riot; when set
         # the "app_name" setting is ignored.
+        #
+        # If your SMTP server requires authentication, the smtp_user & smtp_pass
+        # variables are available.
+        #
         #email:
         #   enable_notifs: false
         #   smtp_host: "localhost"
         #   smtp_port: 25
+        #   require_transport_security: False
```

Comment:
Need examples for smtp_user and smtp_pass too (this way they're in the template and it's obvious that they're options)